lighthouse-extension/app/manifest.json

```diff
@@ -1,12 +1,12 @@
 {
   "name": "__MSG_appName__",
-  "version": "1.6.0",
+  "version": "2.0.0.1",
```

Comment:
Anyway, this version doesn't really matter in the grand scheme of things. It's just for auto updating.